pre-docs/inventory_file.md

```diff
@@ -62,11 +84,21 @@ Config
 A config is a map that contains transport specific configuration options.
 
 Group
-A group is a map that requires a name and can contain a nodes : Nodes object
-and/or a config : Config object. A group name must match the regular expression
-values /[a-zA-Z]\w+/. These are the same values used for environments.  Groups
+A group is a map that requires a name and can contain any of the following:
+- `nodes` : Nodes object
+- `config` : Config object.
+- `facts` : Facts object.
+- `vals` : Vals object.
+
+A group name must match the regular expression
+values /[a-zA-Z]\w+/. This is the same restriction used for environments.
+
+Groups
 An array of group objects.
 
+Facts
```

Comment:
All these sections lack formatting. Do you care? I'd opt for bolding them, or making them headers. Right now they'll all be single paragraphs after markdown formatting.

Reply:
it's a definition list on the site. We can bold them here but lets do that separately from actual docs changes.